lib/config.ts

```diff
@@ -0,0 +1,7 @@
+// This can be used both client and server.
+export const config = {
+  supabase: {
+    url: process.env.NEXT_PUBLIC_SUPABASE_URL!,
+    key: process.env.NEXT_PUBLIC_SUPABASE_KEY!,
```

Comment:
```suggestion
    key: process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY!,
```

see: https://supabase.com/docs/guides/integrations/vercel#step-5-install-supabasejs